src/Tools/Tools.jsx

```diff
@@ -96,6 +96,11 @@ class Tools extends Component {
   };
 
   setActiveTabIndex = (index) => {
+    //Reset error message panel
+    store.dispatch(visualizationSlice.actions.setError(null));
```

Comment:
There is some weird behaviour with displaying the error message, specifically on the first load of the app. The error message with unavailable datasources is present for a split second and then it dissappears (not completely sure what causes it).

Reply:
Means it appears briefly and is immediately hidden?

Reply:
Yes, in the Visualize tab on the first load

Reply:
hm. This does not occur on my browsers, means i need to reproduce it somehow.

Reply:
So for you the error is always visible?

Reply:
No. It works normal for me. Let me know if with this last commit the issue is fixed.

Reply:
Seems to be fixed 👍